core/src/org/pentaho/di/core/exception/IdNotFoundException.java

```diff
@@ -0,0 +1,103 @@
+/*! ******************************************************************************
```

Comment:
This should be a simple unit test to write. Using each of the constructors, I want to verify a get an Exception object that is what I expect. (When I provide a message, make sure I get the same message back. Same for other fields). What is the expected behavior when I pass a null in for any/all the fields? Does it actually behave that way.

Again, should be simple because this is an Exception class and it is just a bean. But there are lines of code that we can cover with unit tests that will help improve our project-level unit test line coverage.